.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E501
```

Comment:
If we could configure the check to limit the line length to 120 chars, I would actually leave it on.
But without that, disabling it completely indeed avoids a lot of annoyance. Screens are larger than 80x25 in this decade (and the previous one).